sys-fs/bees/bees-0.9.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8!

Reply:
Oh, I didn't notice, "pkgcheck" didn't tell me. Do you want to change that during merge, or should I push a revised PR?

Reply:
I've already updated the EAPI locally, will be fine after I push. Thanks!